### ISSUE TITLE
[Training time and anchors depth]

### ISSUE THREAD

**Issue by theodupuis** [MIC-DKFZ/nnDetection#28]
### Question
Two questions : 
- The training is supposed to last for 5 day (more or less) but training the retina-u-net on my data the training for -epochs : 100, batch size : 10, iterations per batch : 200- takes only 8 hours. The nbr of total iterations 'epoch x iterations per batch x batch size' is only " times the total number of iteratons of my training with retina-u-net. Would you have any idea of the things the could explain this time difference ? 
( I checked that the GPU was well used and that the problem wasn't coming from there ) 

- The planner gives as output : 
planner_id': 'D3V001', 'network_dim': 3, 'dataloader_kwargs': {}, 'data_identifier': 'D3V001_3d', 'postprocessing': {}, 'patch_size': array([ 64, 128, 128]), 'batch_size': 4, 'architecture': {'arch_name': 'RetinaUNetV001', 'max_channels': 320, 'start_channels': 32, 'fpn_channels': 128, 'head_channels': 128, 'classifier_classes': 2, 'seg_classes': 2, 'in_channels': 1, 'dim': 3, 'class_weight': [0.3333333333333333, 0.6666666666666667, 0.0], 'conv_kernels': [[1, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3], [3, 3, 3]], 'strides': [[1, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2], [2, 2, 2]], 'decoder_levels': (2, 3, 4, 5)}, 'anchors': {**'width': [(6.0, 4.0, 26.0), (12.0, 8.0, 52.0), (24.0, 16.0, 104.0), (48.0, 32.0, 208.0)],** 'height': [(8.0, 11.0, 15.0), (16.0, 22.0, 30.0), (32.0, 44.0, 60.0), (64.0, 88.0, 120.0)], 'depth': [(8.0, 11.0, 15.0), (16.0, 22.0, 30.0), (32.0, 44.0, 60.0), (64.0, 88.0, 120.0)], 'stride': 1}, 'target_spacing_transposed': array([2. , 0.8125, 0.8125]), 'median_shape_transposed': array([311.875, 512. , 512. ]), 'do_dummy_2D_data_aug': False, 'trigger_lr1': True}
the anchors depth seems strange as the three values are not increasing ... is that a choice from the optimiser that is normal or might it be an error that occured somewhere ? 

Thank you in advance
Best regards
